tests/devices/qubit_mixed/conftest.py

```diff
@@ -42,3 +42,13 @@ def get_random_mixed_state(num_qubits):
 @pytest.fixture(scope="package")
 def random_mixed_state():
     return get_random_mixed_state
+
+
+@pytest.fixture(scope="package")
+def two_qubit_state():
+    return get_random_mixed_state(2)
+
+
+@pytest.fixture(scope="package")
+def two_qubit_batched_state():
+    return np.array([get_random_mixed_state(2) for _ in range(2)])
```

Comment:
You're not using the `random_mixed_state` fixture anywhere.
```suggestion
@pytest.fixture(scope="package")
def two_qubit_state():
    return get_random_mixed_state(2)


@pytest.fixture(scope="package")
def two_qubit_batched_state():
    return np.array([get_random_mixed_state(2) for _ in range(2)])
```

Reply:
Sorry for confusion! I forgot to let you know that last Fri to avoid more unnecessary modification in the future I just pushed the usage fo `random_mixed_state` into master first as it is a modification with other parts which were already finished before; this fixture is used by tests of apply operation.